### DISCUSSION TITLE
Connection to fluidnc through wifi

### DISCUSSION THREAD

**Discussion by pagabee** [winder/Universal-G-Code-Sender#1833] [Help] [answered]
It is possible to connect to the fluidnc through wifi?

There are TCP and WebSocket in the list of connection drivers.
How do these work?
![connection driver](https://user-images.githubusercontent.com/82469736/159171386-50ccae9f-a5f8-4dc3-93e5-06f8a76a4d91.png)

**Comment by breiler** [comment, marked as answer]
The usage for TCP communication is a bit unintuitive. There is a short description for it here:
https://github.com/winder/Universal-G-Code-Sender/wiki/Connecting-the-Controller#smoothie--network

1. Tools -> Options.
1. Click UGS.
1. Open the Sender Options tab.
1. For Connection Driver choose TCPDriver and click OK.
1. Back in the main page, under Firmware at the top choose Smoothie.
1. For Port, type the IP address or hostname of your fluidnc board.
1. For Baud type 23.

**Comment by pagabee** [comment]
Thank you for your support, it works!
